dev/docker/ranger/Dockerfile

```diff
@@ -59,20 +25,18 @@ RUN ARCH=$(uname -m) && \
     ln -s /usr/lib/jvm/java-11-openjdk-amd64 ${JAVA_HOME}; \
     fi
 
-COPY --from=compile-ranger /root/apache-ranger/target/ranger-${RANGER_VERSION}-admin.tar.gz /opt
-RUN cd /opt && \
-    tar zxvf ranger-${RANGER_VERSION}-admin.tar.gz && \
-    ln -s ranger-${RANGER_VERSION}-admin ranger-admin
+# install ranger
+ENV RANGER_HOME=/opt/ranger-admin
+RUN mkdir -p ${RANGER_HOME}
+ADD packages/ranger-${RANGER_VERSION}-admin.tar.gz /opt/
+RUN ln -s /opt/ranger-${RANGER_VERSION}-admin/* ${RANGER_HOME}
 
 # Initialize Ranger envirioment
-RUN curl -L https://search.maven.org/remotecontent?filepath=mysql/mysql-connector-java/8.0.28/mysql-connector-java-8.0.28.jar --output /opt/ranger-admin/ews/webapp/WEB-INF/lib/mysql-connector-java-8.0.28.jar && \
-    cp /opt/ranger-admin/ews/webapp/WEB-INF/lib/mysql-connector-java-8.0.28.jar /opt/ranger-admin/jisql/lib/ && \
-    curl -L https://repo1.maven.org/maven2/com/googlecode/log4jdbc/log4jdbc/1.2/log4jdbc-1.2.jar --output /opt/ranger-admin/ews/webapp/WEB-INF/lib/log4jdbc-1.2.jar && \
-    cp -r /opt/ranger-admin/ews/webapp/WEB-INF/classes/conf.dist/ /opt/ranger-admin/ews/webapp/WEB-INF/classes/conf && \
-    mkdir /opt/ranger-admin/ews/logs
-
-# Clean up
-RUN rm -rf /var/lib/apt/lists/*
+COPY packages/${MYSQL_CONNECTOR_PACKAGE_NAME} ${RANGER_HOME}/ews/webapp/WEB-INF/lib/
+COPY packages/${MYSQL_CONNECTOR_PACKAGE_NAME} ${RANGER_HOME}/jisql/lib/
+COPY packages/${LOG4JDBC_PACKAGE_NAME} ${RANGER_HOME}/ews/webapp/WEB-INF/lib/
+RUN cp -r ${RANGER_HOME}/ews/webapp/WEB-INF/classes/conf.dist/ ${RANGER_HOME}/ews/webapp/WEB-INF/classes/conf && \
+    mkdir ${RANGER_HOME}/ews/logs
```

Comment:
I think better to add the some comments to introduce why we need to copy these JAR in these directories.

Reply:
added.